packages/analytics/analytics-chart/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@kong-ui-public/analytics-chart",
+  "version": "0.8.6",
```

Comment:
I think this is fine, but we may need to watch the CI on release to ensure it bumps the version from this starting point if that's what is intended

Reply:
ahh should I just start it from 0.0.1 ?

Reply:
Since it's a new package name, you could. Alternatively, maybe bump it to `0.9.0` so you can tell the difference.